src/views/Layout.jsx

```diff
@@ -24,17 +24,32 @@ export function Layout() {
 					<Outlet />
 				</main>
 				<nav className="Nav">
-					{!!user ? <SignOutButton /> : <SignInButton />}
 					<div className="Nav-container">
-						<button href="#" className="Nav-link">
+						{!!user ? <SignOutButton /> : <SignInButton />}
+						<NavLink
+							to="/"
```

Comment:
Totally valid, however this can be as simple as something like `<NavLink to="/" className="Nav-link">Home</NavLink>`
Was there a specific reason to use active, pending, transitioning?

Reply:
Honestly the pending and transitioning not really 😅. I was looking at all those functionailties while making sure the NavLink works and I added them.

There was an active class to be set to the buttons when they were active originally, it bolds the underline of the word when that is the acttive class, so to keep that functionality working as I beleive was the goal the active would need to be there.

Reply:
But i did just remember i think active is added by default with NavLink so if we don't have the pending and transition we may not need it specifically set.